lk2nd/project/lk2nd.mk

```diff
@@ -36,3 +36,7 @@ DEFINES := $(filter-out ENABLE_XPU_VIOLATION=1, $(DEFINES))
 # Build Android boot image
 OUTBOOTIMG := $(BUILDDIR)/lk2nd.img
 MKBOOTIMG_CMDLINE := lk2nd
+
+SIGN_BOOTIMG ?= 0
+BOOTIMG_CERT ?= lk2nd/certs/verity.x509.pem
+BOOTIMG_KEY ?= lk2nd/certs/verity.pk8
```

Comment:
Nice! Could you write a short documentation about this in building.md + maybe in a separate commit update the thinksmart comment to explain what `make` line one should use to get a bootable image?

Reply:
Thanks! I've incorporated your suggestions in the latest push.